torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2670,7 +2719,7 @@ def produce_guards(
         source_ref=lambda n: n.name(),
         *,
         input_contexts: Optional[DimList[SymbolicContext]] = None,
-        equalities_inputs: Optional[Set[Tuple[Source, Source]]] = None,
+        equalities_inputs: Optional[EqualityConstraint] = None,
```

Comment:
Could we get some docs for these inputs?

Reply:
Should EqualityConstraint be perhaps pluralized here? It's a little strange that now there is only one equality constraint, seemingly.

Reply:
Yeah sure (for both).